generate-stackbrew-library.sh

```diff
@@ -2,8 +2,8 @@
 set -Eeuo pipefail
 
 declare -A release_channel=(
-	[stable]='18.0.6'
-	[production]='17.0.7'
+	[stable]='19.0.1'
```

Comment:
the stable channel is still on 18, you can see here that 18 instances only get the `18.0.7` update and not `19.0.1`
https://github.com/nextcloud/updater_server/blob/aad2440bcec4af6d19c3cae5993868dea8d8da64/config/config.php#L403-L405
```suggestion
	[stable]='18.0.7'
```

Reply:
Ah, my bad, sorry. Mis-interpreted the config file thinking that since 19.0.1 was *added* (as a whole) in the commit I've referenced above, that this now means that instances will be offered the upgrade to 19.

I've now adjusted the versions to 18.0.7 and 17.0.8, respectively.